laurent22/joplin
# Chore: Mobile: Migrate `global-style.js` to TypeScript

# Summary

This pull request migrates `global-style.js` to TypeScript. This migration was originally done as a part of the mobile plugins pull request (#9652).

# Testing

The goal of this testing plan is to verify that the modified screens that are still `.js` files can open and be used.

1. Open the app
2. Navigate to settings
3. Navigate to the Dropbox login page
4. Navigate to the OneDrive login page
5. Open a note in the editor
6. Add text to the end of the note
7. Open the link dialog
8. Add a drawing
9. Navigate to the "tags" dialog and open a note
10. Show the sidemenu for that note
11. Open the "sync status" screen from settings

This has been successfully tested on Android 13.
